tests/test.rs

```diff
@@ -89,7 +89,7 @@ fn test_unix_connection() {
     let unix_socket_directory = unix_socket_directories.split(',').next().unwrap();
 
     let path = url::percent_encoding::utf8_percent_encode(
-        unix_socket_directory, url::percent_encoding::USERNAME_ENCODE_SET);
```

Comment:
`USERNAME_ENCODE_SET`, `PASSWORD_ENCODE_SET`, `USERINFO_ENCODE_SET` all got merged into `USERINFO_ENCODE_SET` (they were all the same presumably)